tests/core_tests/block_validation.cpp

```diff
@@ -413,7 +380,7 @@ bool gen_block_miner_tx_out_is_small::generate(std::vector<test_event_entry>& ev
   BLOCK_VALIDATION_INIT_GENERATE();
 
   MAKE_MINER_TX_MANUALLY(miner_tx, blk_0);
-  miner_tx.vout[0].amount /= 2;
+  miner_tx.vout[0].amount *= 2;
 
```

Comment:
What is this testing that you changed it from /=2 to *=2? gen_block_miner_tx_out_is_big::generate seems to cover this *=2 scenario.

Reply:
Good catch, should be `/`. However, the `gen_block_miner_tx_out_is_small` doesn't make sense in loki and should be skipped. "From hard fork 2, we allow a miner to claim less block reward than is allowed".

Reply:
So i.e. a miners reward can never be to small? So this test only applies from hardfork < 2? If thats the case, we can delete this test case and once thats done we can merge this in.